lms/djangoapps/learner_dashboard/views.py

```diff
@@ -77,11 +78,19 @@ def program_details(request, program_uuid):
 
     if waffle.switch_is_active('new_program_progress'):
         course_progress = meter.progress(programs=[program_data], count_only=False)[0]
+
+        sidebar_data = {
+            # TODO: Reuse course_progress?
+            'progress': meter.progress(programs=[program_data])[0]
```

Comment:
@AlasdairSwan `course_progress` contains a list of serialized courses in each group (i.e., in progress, completed, not started). `sidebar_data.progress` only contains counts of the courses in each group.

This looks the way it does now because I was sticking to the structure we laid out earlier, but we can split it up easily. We can include separate `program_data`, `course_progress`, and `certificates` objects on the page, though that does mean the JS will need to count the elements in each list contained in `course_progress`. If that's fine, give me a green light and I'll make the change.